packages/libs/src/sdk/api/users/UsersApi.ts

```diff
@@ -416,4 +420,53 @@ export class UsersApi extends GeneratedUsersApi {
     })
     return response.blob()
   }
+
+  async sendTip(request: SendTipRequest) {
+    const { amount } = await parseParams('sendTip', SendTipSchema)(request)
+
+    const getWalletAndUserBank = async (id: string) => {
```

Comment:
Just curious why this is embedded inside `sendTip`? It doesn't seem to reference anything from closure, so I don't think it needs to be created every time this function is called.

Reply:
good point, will clean